packages/backend/src/services/validation.service.ts

```diff
@@ -351,7 +351,15 @@ export default class ValidationService {
       userQuizState.spamFlags > 0 &&
       this.openishStates.includes(quizAnswer.status)
     ) {
-      quizAnswer.status = "manual-review"
+      if (userQuizState.spamFlags >= 3) {
+        if (quizAnswer.status === "enough-received-but-not-given") {
+          quizAnswer.status = "manual-review-once-given-enough"
```

Comment:
This should also catch status "submitted"

Reply:
Added a condition for status "submitted". I assumed the quizAnswer.status is then set to "submitted". Or should I add another status to the enum for it?

Reply:
Sorry I was kinda vague. I meant that status "submitted" should get the same treatment as "enough-received-but-not-given". So it should be:

if (
   quizAnswer.status === "enough-received-but-not-given" || 
   quizAnswer.status ===  "submitted
) {
   quizAnswer.status = "manual-review-once-given-enough"
}

Also, I guess "submitted" should be added to openishStates because it might be set to that on row 306.

This is kind of a mess, but let's just go with it for now :D

Reply:
No worries, fixed the "submitted" conditional. 
I noticed "submitted" was already in the openishStates object (line 277).

Reply:
Äh, sorry. I meant that 'manual-review' should be in the openish states.

Reply:
Added manual-review to openish states.